tests/test-sources/modules/commands.nix

```diff
@@ -0,0 +1,10 @@
+{
+  example = {
+    userCommands = {
+      "W" = {
+        command = ":W<CR>";
```

Comment:
Should this be `:w<CR>` instead of `:W<CR>`?

Reply:
Oh yeah, doesn't really matter since it's the test but I'll change it.